### PR TITLE
Added Bellman_Ford in C

### PR DESCRIPTION
**Pull request by varishtsg**
Values are hard coded in the program. Issue #134

### CODE REVIEW

File: code/graph-algorithms/bellman_ford_algorithm/bellman_ford.c [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#include <stdio.h>
```
Comment (arnavb) [inline code/graph-algorithms/bellman_ford_algorithm/bellman_ford.c]: Could you fix your indentation?

Comment (varishtsg) [reply code/graph-algorithms/bellman_ford_algorithm/bellman_ford.c]: Fixed. Please check.

**Comment by arnavb** [comment]
Yeah, its good now 👍

**Comment by AdiChat** [comment]
Looks good 👍 

Hope you are enjoying your journey with **OpenGenus**

**Pull request merged**